kong/globalpatches.lua

```diff
@@ -15,8 +17,12 @@ return function(options)
   local meta = require "kong.meta"
 
 
-  local cjson = require("cjson.safe")
+  local cjson = require("cjson")
```

Comment:
Do we need this?

Reply:
Already had a private chat and decided to change the PG-related calls to `cjson.safe`